src/Pages/Account/Account.js

```diff
@@ -1,8 +1,229 @@
 import React, { Component } from "react";
+import "../Account/Account.scss";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCheck, faTimes } from "@fortawesome/free-solid-svg-icons";
 
 class Account extends Component {
+  state = {
+    email: "",
+    password: "",
+    nickname: "",
+    isChecked: false,
+    checkId: false,
+    checkPassword: false,
+    checkNickName: false,
+  };
+  // 백엔드에게 회원가입 정보 주기
+  // postSignUp = (nickname, email, password) => {
+  //   fetch("http://192.168.219.191:8000/data/data.json", {
+  //     method: "POST",
+  //     headers: {
+  //       "Content-Type": "data.json",
+  //     },
+  //     body: JSON.stringify({
+  //       email: email,
+  //       password: password,
+  //       nickname: nickname,
+  //     }),
+  //   })
+  //     .then((res) => {
+  //       console.log(res);
+  //     })
+  //     .catch((error) => {
+  //       console.log("통신불가");
+  //     });
+  // };
+
+  // fetchInfo = () => {
+  //   const { isChecked, checkId, checkNickName, checkPassword } = this.state;
+  //   if (isChecked && checkId && checkPassword && checkNickName) {
+  //     this.postSignUp();
+  //   }
+  //   // postSignUp();
+  // };
+
+  handleInput = (e) => {
+    const { value, name } = e.target;
+    this.setState(
+      {
+        [name]: value,
+      },
+      () => this.isVaildationIdPass()
+    );
+  };
+
+  isVaildationIdPass = () => {
+    const { email, password, nickname } = this.state;
+    if (email.includes("@")) {
+      this.setState({ checkId: true });
+    }
+    if (password.length >= 6) {
+      this.setState({ checkPassword: true });
+    }
+    if (nickname.length >= 5) {
+      this.setState({ checkNickName: true });
+    }
+  };
```

Comment:
```suggestion
  isVaildationIdPass = () => {
      const { email, password, nickname } = this.state;
      this.setState({
        checkId: email.includes("@"),
        checkPassword: password.length >= 6,
        checkNickName: nickname.length >= 5
      });
  };
```
- 불리언 데이터 타입을 직접 활용해볼 수 있겠네요👐